### PR TITLE
Adding Virginia to team index

### PR DESCRIPTION
**Pull request by virginiaulrich**
It suggested you as a reviewer, so I'm doing it -- thank you, Owen!

### CODE REVIEW

File: company/team/index.md [hunk 1]
```diff
@@ -473,6 +473,15 @@ To add yourself to this page, copy the following template, paste it at the end o
...
 
+## Virginia Ulrich (she/her)
+
+- Head of Customer Support
+- Portland, Oregon, United States 🇺🇸
+- GitHub: virginiaulrich
+- [virginia@sourcegraph.com](mailto:virginia@sourcegraph.com), [LinkedIn](https://www.linkedin.com/in/virginiaulrich/) 
+- Name pronunciation: vr ji nyuh  oo l ri ch
+- Virginia lives with her partner (Vinh) and two cats, Little Miss Gable (tuxedo) and Hercules Mulligan (orange tabby). She derives joy from the tiny beautiful moments of human connection, hikes across the city and out in nature, and cooking for others. She studied literature and discovered her love of computer science quite by accident. With many yaers in customer service in various forms, Virginia can geek out for hours about the philosophy of 👍 customer support, the importance of compassion > empathy, and all things involving people helping other people.
```
Comment (scalabilitysolved) [inline company/team/index.md:483]: Typo in years `yaers`.  Unless this is specific Portland slang?

**Review by scalabilitysolved — APPROVED** [review]
Welcome to the team @virginiaulrich !  One small typo but otherwise :man_dancing:

File: company/team/index.md (hunk 1 above)
Comment (virginiaulrich) [reply company/team/index.md:483]: Ha! Thank you!

Comment (virginiaulrich) [reply company/team/index.md]: I updated it :)

**Pull request merged**